Animations/README.md

```diff
@@ -0,0 +1,124 @@
+# Animations
+
+[RecyclerViewExtensions](https://github.com/Doist/RecyclerViewExtensions) module providing utilities for animations.
+
+## `AnimatedAdapter`
+
+`Adapter` implementation that provides an `animateDataSetChanges()` method to animate between two data sets. It uses `DataSetDiffer`.
+
+To animate data set changes, `AnimatedAdapter` requires the following method implementations:
```

Comment:
We could be more explicit about the stable ids requirement and mention it here instead of in `getItemId`, e.g.:

> To animate data set changes, `AnimatedAdapter` requires [stable ids](https://developer.android.com/reference/androidx/recyclerview/widget/RecyclerView.Adapter.html#hasStableIds()) and the following method implementations: